lms/djangoapps/discussion/django_comment_client/tests/test_utils.py

```diff
@@ -1015,70 +1011,6 @@ def test_sort_intermediates(self):
             }
         )
 
-    @pytest.mark.skipif(
-        sys.version_info >= (3, 7),
-        reason="Python 3.7 sorted dict insertion is not considered"
-    )
-    def test_sort_intermediates(self):
```

Comment:
why are we removing the whole test? instead of just removing the decorator?

Reply:
We had two tests with the same name and different implementations with different headers. After upgrading we need only one.

Reply:
so these were an identical set of tests for two different python versions?

Reply:
Yes.